generators/entity-client/templates/angular/src/main/webapp/app/entities/_entity.model.ts

```diff
@@ -70,8 +70,29 @@ relationships.forEach(relationship => {
             }
             fieldName = relationship.relationshipFieldName;
         } else {
-            fieldType = tsKeyType;
-            fieldName = `${relationship.relationshipFieldName}Id`;
+            const otherEntityRelationshipName = relationship.otherEntityRelationshipName;
+            const relationshipFieldName = relationship.relationshipFieldName;
+            const relationshipFieldNamePlural = relationship.relationshipFieldNamePlural;
+            const relationshipType = relationship.relationshipType;
+            const otherEntityNameCapitalized = relationship.otherEntityNameCapitalized;
+            const otherEntityFieldCapitalized = relationship.otherEntityFieldCapitalized;
+            const ownerSide = relationship.ownerSide;
+
+            if (relationshipType === 'many-to-many' && ownerSide === true) {
+                fieldType = otherEntityNameCapitalized + '[]';
```

Comment:
minor nit you could do string literal here `${otherEntityNameCapitalized}[]`

Reply:
Agree